dev-build/make/make-9999.ebuild

```diff
@@ -32,8 +32,11 @@ RDEPEND="
 BDEPEND="
 	nls? ( sys-devel/gettext )
 	verify-sig? ( sec-keys/openpgp-keys-make )
+	doc? ( sys-apps/texinfo )
 "
 
+DOCS="AUTHORS NEWS README*"
```

Comment:
Can we change to array please?

Reply:
I haven't been able to successfully set DOCS=() with the wildcard README contained within it.